tests/local/systemreports/records_test.php

```diff
@@ -40,7 +40,7 @@ public function test_records() {
             'submissionid' => '2',
             'reviewerid' => '2',
             'weight' => '1',
-            'feedbackauthor' => '<p>Bad data &lt;img alt="" src="data:image/gif;base64,R0lGODdhAQABAPAAAP8AAAAAACwAAAAAAQABAAACAkQBADs=" /&gt;</p>',
```

Comment:
given this is a unit test I think this one is better hard coded than in a string which could change